PhotoKingdomAPI/PhotoKingdomAPI/Controllers/ViewModels/Resident_vm.cs

```diff
@@ -1,98 +1,109 @@
+			Pings = new List<PingBase>();
+			AttractionPhotowarVotes = new List<AttractionPhotowarUploadBase>();
+			CountryPhotowarVotes = new List<CountryPhotowarUploadBase>();
+			ContinentPhotowarVotes = new List<ContinentPhotowarUploadBase>();
+			ResidentAttractionOwns = new List<ResidentAttractionOwnBase>();
+			ResidentCityOwns = new List<ResidentCityOwnBase>();
+			ResidentProvinceOwns = new List<ResidentProvinceOwnBase>();
+			ResidentCountryOwns = new List<ResidentCountryOwnBase>();
+			ResidentContinentOwns = new List<ResidentContinentOwnBase>();
+			RequestedCountryPhotowarPhotorequests = new List<CountryPhotowarPhotorequestBase>();
+			ReceivedCountryPhotowarPhotorequests = new List<CountryPhotowarPhotorequestBase>();
+			RequestedContinentPhotowarPhotorequests = new List<ContinentPhotowarPhotorequestBase>();
 			ReceivedContinentPhotowarPhotorequests = new List<ContinentPhotowarPhotorequestBase>();
-        }
-		// the city the Resident belongs to
-		public String CityName { get; set; }
-		public String CityProvinceName { get; set; }
-		public String CityCountryName { get; set; }
-
-		// Resident's photos
-		public IEnumerable<PhotoBase> Photos { get; set; }
-
-		// Resident's pings
-		public IEnumerable<PingBase> Pings { get; set; }
-
-		// the votes that the Resident has made
-		public IEnumerable<AttractionPhotowarUploadBase> AttractionPhotowarVotes { get; set; }
-		public IEnumerable<CountryPhotowarUploadBase> CountryPhotowarVotes { get; set; }
-		public IEnumerable<ContinentPhotowarUploadBase> ContinentPhotowarVotes { get; set; }
-
-		// all the "owns" this resident has had
-		public IEnumerable<ResidentAttractionOwnBase> ResidentAttractionOwns { get; set; }
-		public IEnumerable<ResidentCityOwnBase> ResidentCityOwns { get; set; }
-		public IEnumerable<ResidentProvinceOwnBase> ResidentProvinceOwns { get; set; }
-		public IEnumerable<ResidentCountryOwnBase> ResidentCountryOwns { get; set; }
-		public IEnumerable<ResidentContinentOwnBase> ResidentContinentOwns { get; set; }
-
-		// the CountryPhotowarPhotorequests this resident has made to subordinates
-		public IEnumerable<CountryPhotowarPhotorequestBase> RequestedCountryPhotowarPhotorequests { get; set; }
-
-		// the CountryPhotowarPhotorequests this resident has received from the King/Queen
-		public IEnumerable<CountryPhotowarPhotorequestBase> ReceivedCountryPhotowarPhotorequests { get; set; }
-
-		// the ContinentPhotowarPhotorequests this resident has made to subordinates
-		public IEnumerable<ContinentPhotowarPhotorequestBase> RequestedContinentPhotowarPhotorequests { get; set; }
-
-		// the ContinentPhotowarPhotorequests this resident has received from the Emperor/Empress
-		public IEnumerable<ContinentPhotowarPhotorequestBase> ReceivedContinentPhotowarPhotorequests { get; set; }
-
+        }
+		// the city the Resident belongs to
+		public String CityName { get; set; }
+		public String CityProvinceName { get; set; }
+		public String CityCountryName { get; set; }

```

Comment:
we don't have a direct association between City and Country anymore right? It should be CityProvinceCountryName ? I'll still merge but we will change that next commit.

Reply:
Yes, it's right.